store/aciinfo.go

```diff
@@ -21,46 +21,67 @@ import (
 	"time"
 )
 
+// ACIFetchInfo is used to store information about the fetching process of an ACI.
+type ACIFetchInfo struct {
+	// Latest defines if the ACI was imported using the latest pattern
+	// (no version label was provided on ACI discovery).
+	Latest bool
+
+	// VerificationHash is the GPG fingerprint that was used to verify the validity of the ACI.
+	VerificationHash string
+
+	// SourceURL is the URL used to import an ACI.
+	SourceURL string
+
+	// InsecureOptions are the insecure options that was used when retrieving the ACI.
+	InsecureOptions string
+}
+
 // ACIInfo is used to store information about an ACI.
 type ACIInfo struct {
 	// BlobKey is the key in the blob/imageManifest store of the related
 	// ACI file and is the db primary key.
 	BlobKey string
+
 	// Name is the name of the ACI.
 	Name string
+
 	// ImportTime is the time this ACI was imported in the store.
 	ImportTime time.Time
-	// LastUsed is the last time this image was read
+
+	// LastUsed is the last time this image was read.
 	LastUsed time.Time
-	// Size is the size in bytes of this image in the store
+
+	// Latest defines if the ACI was imported using the latest pattern
+	// (no version label was provided on ACI discovery).
+	Latest bool
+
+	// Size is the size in bytes of this image in the store.
 	Size int64
+
 	// TreeStoreSize is the size in bytes of this image in the tree store
 	TreeStoreSize int64
-	// Latest defines if the ACI was imported using the latest pattern (no
-	// version label was provided on ACI discovery)
-	Latest bool
-}
 
-func NewACIInfo(blobKey string, latest bool, t time.Time, size int64, treeStoreSize int64) *ACIInfo {
-	return &ACIInfo{
-		BlobKey:       blobKey,
-		Latest:        latest,
-		ImportTime:    t,
-		LastUsed:      time.Now(),
-		Size:          size,
-		TreeStoreSize: treeStoreSize,
-	}
+	// VerificationHash is the GPG fingerprint that was used to verify the validity of the ACI.
+	VerificationHash string
```

Comment:
Am I wrong or the various `*Fetcher` pass the image signature instead of the gpg public key fingerprint?